project.go

```diff
@@ -32,33 +32,35 @@ type (
-		VatID            *string          `json:"vat_id,omitempty"`
+		BillingAddress               *string          `json:"billing_address,omitempty"`
+		BillingEmails                *[]*ContactEmail `json:"billing_emails,omitempty"`
+		BillingExtraText             *string          `json:"billing_extra_text,omitempty"`
+		CardID                       *string          `json:"card_id,omitempty"`
+		Cloud                        *string          `json:"cloud,omitempty"`
+		CopyFromProject              string           `json:"copy_from_project,omitempty"`
+		CountryCode                  *string          `json:"country_code,omitempty"`
+		Project                      string           `json:"project"`
+		AccountId                    *string          `json:"account_id,omitempty"`
+		TechnicalEmails              *[]*ContactEmail `json:"tech_emails,omitempty"`
+		BillingCurrency              string           `json:"billing_currency,omitempty"`
+		VatID                        *string          `json:"vat_id,omitempty"`
+		UseSourceProjectBillingGroup bool             `json:"use_source_project_billing_group,omitempty"`
 	}
 
 	// UpdateProjectRequest are the parameters for updating a project.
 	UpdateProjectRequest struct {
-		Name             string           `json:"project_name,omitempty"`
-		BillingAddress   *string          `json:"billing_address,omitempty"`
-		BillingEmails    *[]*ContactEmail `json:"billing_emails,omitempty"`
-		BillingExtraText *string          `json:"billing_extra_text,omitempty"`
-		CardID           *string          `json:"card_id,omitempty"`
-		Cloud            *string          `json:"cloud,omitempty"`
-		CountryCode      *string          `json:"country_code,omitempty"`
-		AccountId        *string          `json:"account_id,omitempty"`
-		TechnicalEmails  *[]*ContactEmail `json:"tech_emails,omitempty"`
-		BillingCurrency  string           `json:"billing_currency,omitempty"`
-		VatID            *string          `json:"vat_id,omitempty"`
+		Name                         string           `json:"project_name,omitempty"`
+		BillingAddress               *string          `json:"billing_address,omitempty"`
+		BillingEmails                *[]*ContactEmail `json:"billing_emails,omitempty"`
+		BillingExtraText             *string          `json:"billing_extra_text,omitempty"`
+		CardID                       *string          `json:"card_id,omitempty"`
+		Cloud                        *string          `json:"cloud,omitempty"`
+		CountryCode                  *string          `json:"country_code,omitempty"`
+		AccountId                    *string          `json:"account_id,omitempty"`
+		TechnicalEmails              *[]*ContactEmail `json:"tech_emails,omitempty"`
+		BillingCurrency              string           `json:"billing_currency,omitempty"`
+		VatID                        *string          `json:"vat_id,omitempty"`
+		UseSourceProjectBillingGroup bool             `json:"use_source_project_billing_group,omitempty"`
```

Comment:
Project update API does not accept this parameter

Reply:
Ah, right, it is only available when creating a new user.